### ISSUE TITLE
Cannot build w/rake - undefined method 'each' for "i386-mingw32":String

### ISSUE THREAD

**Issue by Ataye** [eventmachine/eventmachine#422]
I cannot get this to build but chances are i'm doing something wrong (and new to ruby)!  I'm on windows 8 64 with:

- ruby 2.0.0p0 (2013-02-24) [x64-mingw32]
- DevKit-mingw64-64-4.7.2-20130224-1432-sfx (have tried mingw64-32)
- latest rake

I cannot `gem install eventmachine` because of `project.h:97:13: error: 'pid_t'` error so i need to build manually.  I have installed rake, cloned Git and issued `rake build --trace` and received (edited):

```
rake aborted!
undefined method 'each' for "i386-mingw32":String
C:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/eventmachine/rakelib/package.rake:34:in
'hack_cross_compilation'
C:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/eventmachine/rakelib/package.rake:44:in
'<top (required)>'
C:/Ruby200-x64/lib/ruby/2.0.0/rake/rake_module.rb:25:in 'load'
C:/Ruby200-x64/lib/ruby/2.0.0/rake/rake_module.rb:25:in 'load_rakefile'
...
```

I have tried to inject devkit functionality into eventmachine but not sure if that's right.  I did get an error about `git ls-files` because git (portable) wasn't in the PATH.

Any help would be greatly appreciated.
James

**Comment by sodabrew** [comment]
This is resolved by eventmachine 1.0.4. Please give it a try and close the issue or comment with follow-up issues. Thanks!

**Comment by Ataye** [comment]
Ok thats working for me now so i'll close it.  Thank's for sorting it :)
